content/03.methods.md

```diff
@@ -51,13 +51,78 @@ Numbers in parentheses show the number of positives and negatives that resulted
 We used these annotations to train a discriminator model that makes the final classification step.
 Our label functions fall into three categories: databases, text patterns and domain heuristics.
-We provide examples for each category in our [supplemental methods section](#label-function-categories).  
+
+#### Label Function Categories
+
+We provide examples for each category below. 
+Each example pertains to the following candidate sentence: “[PTK6]{.gene_color} may be a novel therapeutic target for [pancreatic cancer]{.disease_color}.”
+
+**Databases**: These label functions incorporate existing databases to generate a signal, as seen in distant supervision [@doi:10.3115/1690219.1690287].
+These functions detect if a candidate sentence's co-mention pair is present in a given database.
+If the pair is present, our label function emits a positive label and abstain otherwise.
+If the pair is not present in any existing database, a separate label function emits a negative label.
+We used a separate label function to prevent a label imbalance problem that we encountered during development: emitting positives and negatives from the same label function causes downstream classifiers to generate almost exclusively negative predictions.
+
+$$ \Lambda_{DB}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ 1 & (\color{#875442}{D}, \color{#02b3e4}{G}) \in DB \\
+0 & otherwise \\
+\end{cases} $$
+
+$$ \Lambda_{\neg DB}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ -1 & (\color{#875442}{D}, \color{#02b3e4}{G}) \notin DB \\
+0 & otherwise \\
+\end{cases} $$
+
+**Text Patterns**: These label functions are designed to use keywords and sentence context to generate a signal. 
+For example, a label function could focus on the number of words between two mentions or focus on the grammatical structure of a sentence.
+These functions emit a positive or negative label depending on the context.
+
+$$ \Lambda_{TP}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ 1 & "target" \> \in Candidate \> Sentence \\
+ 0 & otherwise \\
+\end{cases} $$
+
+$$ \Lambda_{TP}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ -1 & 	"VB" \> \notin pos\_tags(Candidate \> Sentence) \\
+ 0 & otherwise \\
+\end{cases} $$
+
+
+**Domain Heuristics**: These label functions use the other experiment results to generate a signal. 
+We used dependency path cluster themes generated by Percha et al. [@tag:global_network].
+If a candidate sentence's dependency path belonged to a previously generated cluster, then the label function will emit a positive label and abstain otherwise.
+
+$$
+\Lambda_{DH}(\color{#875442}{D}, \color{#02b3e4}{G}) = \begin{cases}
+    1 & Candidate \> Sentence \in Cluster \> Theme\\
+    0 & otherwise \\
+    \end{cases}
+$$
+
+Each label function was constructed via manual examination of sentences within the training set.
```

Comment:
This follows from the most recent bold as domain heuristics, but I think it actually applies primarily to text patterns. Have you considered swapping the order of these two categories?